Idoit.API.Client/CMDB/ObjectTypes/ObjectTypes.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Idoit.API.Client.CMDB.ObjectTypes.Response;
+
+namespace Idoit.API.Client.CMDB.ObjectTypes
+{
+    public class ObjectTypes
+    {
+        public ObjectTypes(Client myClient)
+        {
+            client = myClient;
+        }
+
+        Dictionary<string, object> parameter;
+        List<Result[]> response;
+        public Client client;
```

Comment:
I would move the class properties above the constructor for a better overview. But that's just my personal feeling 😄

Reply:
Agree, if there is no C# convention that says otherwise.